observing_examples.ipynb

```diff
@@ -0,0 +1,971 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sdss = ['rprime', 'zprime']  # only want r' and z' bands for now\n",
+    "svo_sdss_link = 'http://svo2.cab.inta-csic.es/theory/fps' + \\\n",
+    "                 '/getdata.php?format=ascii&id=SLOAN/SDSS.'\n",
+    "\n",
+    "bandpasses = {}\n",
+    "for band in sdss:\n",
+    "    local_path = download_file(svo_sdss_link + band + '_filter')\n",
+    "    bp = SpectralElement.from_file(local_path)\n",
+    "    bandpasses[band] = bp\n",
+    "    \n",
+    "    plt.plot(bp.waveset, bp(bp.waveset), label=band)\n",
+    "plt.legend(loc='lower right')\n",
+    "plt.xlim(3000, 14000)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"atmosphere\"></a>\n",
+    "## 5. Model the attenuation by the atmosphere"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Model the sky by querying results from the <a href=\"http://etimecalret-001.eso.org/observing/etc/bin/gen/form?INS.MODE=swspectr+INS.NAME=SKYCALC\">SkyCalc Sky Model Calculator</a>:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# SkyCalc recognizes Cerro Paranal by its id number 2460, which we \n",
+    "# got from the skycalc_cli source code\n",
+    "paranal_id = '2640'\n",
+    "\n",
+    "# Use the default parameters from skycalc to set up the parameter dict\n",
+    "params_path = download_file('http://www.eso.org/observing/etc/' + \\\n",
+    "                                'doc/skycalc/skycalc_defaults.txt')        \n",
+    "\n",
+    "params = {}\n",
+    "with open(params_path) as o:\n",
```

Comment:
Stylistically, I'm wondering if this code should be in the notebook, or sit in a separate `.py` file in the same directory as the notebook, which gets imported and handles the query for you. Ideally moving things into a separate file will help us refine our code before we make a PR to astroquery, and will keep the focus of the notebook on the mechanics of `synphot` rather than `skycalc`. Any thoughts @pllim/@eteq?

Reply:
For fast prototyping, it is more convenient to have everything in a notebook. When this is stable, then we can revisit on what needs to go where properly. tl;dr -- I have no preference at this stage.

Reply:
I agree that it kind of distracts from the point of the notebook, but I'm also not sure if it's better to have a separate file that the user would need.. hmm

Reply:
Yet another alternative is you could have the notebook install `skycalc_cli` with `!pip install skycalc_cli` and then you can use their command-line interface to download the curve directly from the notebook.

Either way, I think it'd be great if you defined a function with a good docstring called `get_atmospheric_transmittance` which contained all of the querying machinery and simply returns the wavelength and transmittance grid. That way it's easy to move if we decide it should be moved later.

Reply:
> then you can use their command-line interface to download the curve directly from the notebook.
This would be the most aesthetically pleasing, but it would require the user to provide a parameters file which could get confusing.

> Either way, I think it'd be great if you defined a function with a good docstring called `get_atmospheric_transmittance`

On second thought I think making another file containing this function would be a good approach, since I will end up needing the same functionality in the other examples.